usage_example.ipynb

```diff
@@ -1,587 +1 @@
-{
```

Comment:
Something weird going on here - all line breaks are gone!

Reply:
Exported again and it seems fine now - very strange!